### PR TITLE
Add missing destructor

### PR DESCRIPTION
**Pull request by kurapov-peter**
Well... happens, I guess.

### CODE REVIEW

File: omniscidb/L0Mgr/L0Mgr.cpp [hunk 1]
```diff
@@ -114,6 +114,10 @@ ze_command_list_handle_t L0CommandList::handle() const {
 
 L0CommandList::~L0CommandList() {
   // TODO: maybe return to pool
+  auto status = zeCommandListDestroy(handle_);
+  if (status) {
+    std::cerr << "Non-zero status for command list destructor" << std::endl;
```
Comment (ienkovich) [inline omniscidb/L0Mgr/L0Mgr.cpp:119]: Why not LOG(ERROR)?

Comment (kurapov-peter) [reply omniscidb/L0Mgr/L0Mgr.cpp:119]: No particular reason, just to stay consistent.

Comment (ienkovich) [reply omniscidb/L0Mgr/L0Mgr.cpp:119]: I see. I still think moving from `std::err` to `LOG(ERROR)` in this file would be the right thing to do. In tests, ERROR log should be printed to stderr anyway.

**Review by ienkovich — APPROVED** [review]

File: omniscidb/L0Mgr/L0Mgr.cpp (hunk 1 above)
Comment (kurapov-peter) [reply omniscidb/L0Mgr/L0Mgr.cpp]: Done

**Pull request merged**